cmd/atlas/atlas.go

```diff
@@ -48,12 +49,17 @@ To learn more, see our documentation: https://www.mongodb.com/docs/atlas/cli/sta
 }
 
 // loadConfig reads in config file and ENV variables if set.
-func loadConfig() error {
-	if err := config.LoadAtlasCLIConfig(); err != nil {
-		return fmt.Errorf("error loading config: %w. Please run `atlas config init` to reconfigure your profile", err)
+func loadConfig() (*config.Profile, error) {
+	configStore, initErr := config.LoadViperStore(afero.NewOsFs())
+
+	if initErr != nil {
+		return nil, fmt.Errorf("error loading config: %w. Please run `atlas config init` to reconfigure your profile", initErr)
```

Comment:
With #4044, 'atlas config init' is deprecated. we should avoid pointing users to it
```suggestion
		return nil, fmt.Errorf("error loading config: %w. Please run `atlas auth login` to reconfigure your profile", initErr)
```

Reply:
Sure, I didn't write this. I moved the line from somewhere else :)